install.m

```diff
@@ -31,7 +33,13 @@ function install(input)
 
     % Install all the packages via conda
     fprintf('Installing packages\n');
-    system(sprintf('"%s" create -n "%s" -y -c conda-forge -c robotology casadi-matlab-bindings=3.5.5.2 "libblas=*=*openblas"\n', mamba_full_path,input.env_name));
+    if input.install_casadi_via_mamba
+        system(sprintf('"%s" create -n "%s" -y -c conda-forge -c robotology casadi-matlab-bindings=3.5.5.2 "libblas=*=*openblas"\n', mamba_full_path,input.env_name));
+    else
+        system(sprintf('"%s" create -n "%s" -y -c conda-forge -c robotology \n', mamba_full_path,input.env_name));
```

Comment:
Why do you need the environment if you install casadi by the official binaries?

Reply:
Because in the environment I also declare the MATLABPATH env variable (see https://github.com/ami-iit/mystica/pull/14/files#diff-54c250a9c215f1709a131ba16c0752bab45e3460408a7ab3152a992dcda74ba3R140).
I usually run matlab from a terminal where the mamba environment is active, in this way it's not necessary to run the autogenerated `setup.m`